tempesta_fw/http_sess.c

```diff
@@ -83,10 +84,10 @@ static TfwCfgSticky tfw_cfg_sticky;
 static struct crypto_shash *tfw_sticky_shash;
 static char tfw_sticky_key[STICKY_KEY_MAXLEN];
 
-SessHashBucket sess_hash[SESS_HASH_SZ] = {
-	[0 ... (SESS_HASH_SZ - 1)] = {
-		HLIST_HEAD_INIT,
-	}
+static SessHashBucket sess_hash[SESS_HASH_SZ] = {
+//	[0 ... (SESS_HASH_SZ - 1)] = {
+//		HLIST_HEAD_INIT,
+//	}
```

Comment:
Didn't want C++ comments to appear here. I use IDE with integrated clang code analysis and clang gets stuck on this initialization and the whole IDE hangs every time i make changes to this file.  I will be twice more careful on sending patches next time.